### PR TITLE
creates /opt/wwc/data

### PR DESCRIPTION
**Pull request by jbau**

### CODE REVIEW

File: playbooks/roles/lms/tasks/main.yml [hunk 1]
```diff
@@ -18,6 +18,11 @@
 - include: ../../nginx/tasks/nginx_site.yml state=link site_name=lms
 - include: ../../nginx/tasks/nginx_site.yml state=link site_name=lms-backend
 
+- name: Change permissions on datadir
+  file: path={{ app_base_dir }}/data state=directory owner=www-data group=www-data
```
Comment (jarv) [inline playbooks/roles/lms/tasks/main.yml:22]: Can we set the permissions to what we want in "Create data dir" and remove this task?

Comment (jrbl) [inline playbooks/roles/lms/tasks/main.yml:22]: I don't think so, because I don't think that the www-data user exists yet
when Common runs.

**Comment by jarv** [comment]
good to merge

**Comment by jrbl** [comment]
I rebased on current master but didn't change anything, so should still be good to merge.

Shipit.

**Comment by jrbl** [comment]
I re-rebased and tested, and added the cms tag to the directory permission update. Merging.

**Pull request merged**